### ISSUE TITLE
Unexpected input in CLI compiler

### ISSUE THREAD

**Issue by oliversalzburg** [messageformat/messageformat#223]
I just installed the CLI compiler and did not get very far with it. It just presents me with this error, that doesn't tell me anything:

```
$ messageformat de-DE.json

C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\node_modules\messageformat-parser\parser.js:1605
    throw peg$buildStructuredError(
    ^
SyntaxError: Expected [\t-\r \x85‎‏  ] or [^\t-\r \x85‎‏  !-/:-@[-\^`{-~¡-§©«¬®°±¶»¿×÷‐-‧‰-‾⁁-⁓⁕-⁞←-⑟─-❵➔-⯿⸀-⹿、-〃〈-〠〰﴾﴿﹅﹆] but "{" found.
    at peg$buildStructuredError (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\node_modules\messageformat-parser\parser.js:368:12)
    at peg$parse (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\node_modules\messageformat-parser\parser.js:1605:11)
    at Compiler.compile (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\lib\compiler.js:157:13)
    at Compiler.compile (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\lib\compiler.js:163:26)
    at Compiler.compile (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\lib\compiler.js:163:26)
    at Compiler.compile (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\lib\compiler.js:163:26)
    at MessageFormat.compile (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat\lib\messageformat.js:339:22)
    at Object.<anonymous> (C:\Users\OliverSalzburg\AppData\Local\nvs\node\10.13.0\x64\node_modules\messageformat-cli\messageformat.js:80:19)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
```

I'm not sure _where_ the `{` is unexpected, but it's a pretty common symbol in a JSON file :D

**Comment by oliversalzburg** [comment]
Yeah, I just arrived at that conclusion too :D I was not aware of all the layers that play a role in our setup. We're using `angular-translate-interpolation-messageformat` and I'm not sure if that applies any transformation to the locale data, but most importantly it relies on messageformat 1.x and I tried the CLI with 2.x

Overall, it's all moot though, because I was trying to resolve the CSP generated by `new Function()`, but `angular-translate-interpolation-messageformat` specifically makes use of the runtime compiler. So any build time compilation efforts would be pointless anyway.

Sadly, I can't share the file, but, given the breaking changes in the release notes, I would assume it's a `{` in a string. I can reproduce the issue with something like this:
```json
{
	"hub": {
		"title": "{{name}} foo"
	}
}
```
